test/utils.cpp

```diff
@@ -814,6 +814,31 @@ TEST_CASE("getcwd() returns current directory of the process", "[utils]")
 		REQUIRE(datadir.substr(datadir.length() - subdir.length()) ==
 			subdir);
 	}
+
+	SECTION("Invalid current directories")
```

Comment:
Can this be made more descriptive? "Returns empty string if current dir doesn't exist", perhaps? "Invalid" can mean a lot of things, and if this test fails, one will have to read the code to find out what exactly does the description mean.

Reply:
Fixed.